modules/cli-options/src/main/scala/scala/cli/commands/PackageOptions.scala

```diff
@@ -62,6 +62,12 @@ final case class PackageOptions(
   @HelpMessage("Build Docker image")
     docker: Boolean = false,
 
+  @Group("Package")
+  @Hidden
+  @HelpMessage("Exclude modules *and their transitive dependencies* from the JAR to be packaged")
+  @ValueDescription("org:name")
+    provided: List[String] = Nil,
```

Comment:
Why not name this parameter as `--exclude`?

Reply:
"Exclude" is already used in [sbt](https://www.scala-sbt.org/1.x/docs/Library-Management.html#Exclude+Transitive+Dependencies), Mill, coursier, …, to exclude a dependency, but not necessarily all of its transitive dependencies. The option here is slightly different.

That said, "provided" is also used in sbt, and has slightly different semantics there (it means "compile-time" somehow). I would argue that the semantics here are closer to what users expect (which would be compile-time too, but also assume the dependency is already around, including all of its transitive dependencies).